bftengine/src/bftengine/ClientsManager.cpp

```diff
@@ -336,8 +336,15 @@ Time ClientsManager::infoOfEarliestPendingRequest(std::string& cid) const {
   RequestInfo earliestPendingReqInfo{MaxTime, std::string()};
   for (const ClientInfo& clientInfo : indexToClientInfo_) {
     for (const auto& req : clientInfo.requestsInfo) {
+      if (req.second.committed) {
+        // It is enough if the client has even a single committed request. In this case, we know that the system is
+        // alive and that we don't need to run a VC.
+        earliestPendingReqInfo = {MaxTime, std::string()};
+        earliestTime = MaxTime;
+        break;
+      }
       // Don't take into account already committed requests
-      if ((req.second.time != MinTime) && (earliestTime > req.second.time) && (!req.second.committed)) {
+      if ((req.second.time != MinTime) && (earliestTime > req.second.time)) {
         earliestPendingReqInfo = req.second;
         earliestTime = earliestPendingReqInfo.time;
       }
```

Comment:
Maybe it worth changing the name of this function to reflect what it actually does now.